app/services/form1010cg/service.rb

```diff
@@ -63,6 +63,9 @@ def process_claim_v2!
       [claim_pdf_path, poa_attachment_path].each { |p| File.delete(p) if p.present? }
 
       CARMA::Client::MuleSoftClient.new.create_submission_v2(payload)
+    rescue => e
+      Rails.logger.info "[Form 10-10CG] Submission failed for claim_guid: #{claim.guid}. Exception: #{e.message}"
```

Comment:
I was mostly thinking in datadog. When a submission fails you can look at the logs there and walk through the steps, and this would be at the end there which I found helpful. Although Sentry also logs to the rails logger, right? I'm still unclear of what our best practices are for Sentry vs Rails logger. Is it just anything _not_ an error should be in the rails logger vs errors go to Sentry?

Reply:
Okay, thanks for the context on that. I think you're right that logging it to sentry makes the most sense here 👍 .